README.md

```diff
@@ -12,15 +12,18 @@ go-buffer-pool
 
 ## Table of Contents
 
-- [Use Case](#use-case)
+- [About](#about)
     - [Advantages over GC](#advantages-over-gc)
     - [Disadvantages over GC:](#disadvantages-over-gc)
 - [Contribute](#contribute)
 - [License](#license)
 
-## Use Case
+## About
 
-Use this when you need to repeatedly allocate and free a bunch of temporary buffers of approximately the same size.
+This library provides:
+
+1. A buffer pool for re-using byte buffers of _varying_ sizes. Unlike `sync.Pool`, this pool will always return an appropriately sized buffer (the next power of two) so it's appropriate for applications that need to pool dynamically sized buffers.
```

Comment:
This is confusing to me because I think you're referring to a byte slice — `[]byte` — as a byte buffer.

(It's also confusing to me why this property is important, but I guess that's a separate discussion.)

Reply:
I see. Often you will find [sync.Pools segregated by size class](https://github.com/golang/go/blob/7e394a2/src/net/http/h2_bundle.go#L998-L1043) as a solution to this problem, too.

Reply:
Yes, that's what we're doing here. But the code you linked will cause allocations (because casting a slice to an interface will allocate). That's why we have a nice library to do this for us.